.eslintrc.yml

```diff
@@ -0,0 +1,15 @@
+root: true
+env:
+  browser: true
+  jest/globals: true
+extends:
+  - eslint:recommended
+  - prettier # eslint-config-prettier
+  - plugin:@typescript-eslint/recommended
+plugins:
+  - jest
+rules:
+  '@typescript-eslint/ban-ts-comment': off # temporary disable while addressing existing violations
+  '@typescript-eslint/consistent-type-exports': error
+  '@typescript-eslint/consistent-type-imports': error
```

Comment:
Motivation: https://typescript-eslint.io/blog/consistent-type-imports-and-exports-why-and-how